### PR TITLE
Build warnings: dept features, news

### PR DESCRIPTION
**Pull request by Garneauma**
Resolving build warnings part 8:
- Adding gc-features sample file
- Fixing news templates

### CODE REVIEW

File: _includes/components/gc-features/features.html [hunk 1]
```diff
@@ -0,0 +1,31 @@
+{%- comment -%}
+
+++++++++++++++++++++++++++++++++++
++ List of features
+++++++++++++++++++++++++++++++++++
+
```
Comment (duboisp) [inline _includes/components/gc-features/features.html:6]: This file seem to be a duplicate of "components/gc-features/samples"

File: templates/gc-audience-en.html [hunk 2]
```diff
@@ -19,4 +19,4 @@
 	</div>
 </div>
 {% include components/gc-servinfo/2col.html %}
-{{>deptfeatures}}
+{% include components/gc-features/features.html %}
```
Comment (duboisp) [inline templates/gc-audience-en.html:22]: Use include relative for content sample

File: components/gc-features/samples/features.html [hunk 3]
```diff
@@ -0,0 +1,31 @@
+{%- comment -%}
+
+++++++++++++++++++++++++++++++++++
++ List of features
+++++++++++++++++++++++++++++++++++
+
+{%- endcomment -%}
+
+<section class="gc-features">
```
Comment (duboisp) [inline components/gc-features/samples/features.html:9]: Side note: the gc-features is also about the use of this class "gc-prtts" as showed in the news-en.html here: https://github.com/wet-boew/themes-dist/blob/v4.0.28-gcweb/news-en.html#L205-L208

**Review by duboisp — CHANGES_REQUESTED** [review]
* [ ] Remove duplication in "_includes" folder
* [ ] Use include relative for content sample

Note: the news template seems to have a panel aside, don't know exactly what it do but we can see the code here: https://github.com/wet-boew/themes-dist/blob/v4.0.28-gcweb/news-en.html#L104-L153

Side note: the "gc-features" is also about the use of the class "gc-prtts" as showed in [news-en.html](https://github.com/wet-boew/themes-dist/blob/v4.0.28-gcweb/news-en.html#L205-L208)

File: _includes/components/gc-features/features.html (hunk 1 above)
Comment (Garneauma) [reply _includes/components/gc-features/features.html:6]: It is, the `grunt` task copies the file. I can remove it, but anyone running `grunt` would have it copied.

File: templates/gc-audience-en.html (hunk 2 above)
Comment (Garneauma) [reply templates/gc-audience-en.html:22]: As discussed [here](https://github.com/wet-boew/GCWeb/pull/2089#issuecomment-1402109378), include_relative wil not work in this case.

**Comment by Garneauma** [comment]
> * [ ]  Remove duplication in "_includes" folder
> * [ ]  Use include relative for content sample
> 
> Note: the news template seems to have a panel aside, don't know exactly what it do but we can see the code here: https://github.com/wet-boew/themes-dist/blob/v4.0.28-gcweb/news-en.html#L104-L153
> 
> Side note: the "gc-features" is also about the use of the class "gc-prtts" as showed in [news-en.html](https://github.com/wet-boew/themes-dist/blob/v4.0.28-gcweb/news-en.html#L205-L208)

No update needed. Comments addressed individuallly.

**Comment by duboisp** [comment]
Thanks @Garneauma 

Before to forget about it, I would like to address this missing/undocumented content: https://github.com/wet-boew/themes-dist/blob/v4.0.28-gcweb/news-en.html#L104-L153
It is outside the content page area and the corresponding CSS was assigned by mistake as a tab interface improvement which is not. I do know that pattern should not used anymore since we went with the un-carrousel + randomizer plugin as a promotional replacement a few year ago. I will like us to dispose of that custom promotion (`prm-flpr`) CSS according to our new guidance. Let's chat about it.

File: templates/news/news-doc-en.html [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+
+<h2>Working examples</h2>
+<ul>
+	<li><a href="news-en.html">Working example</a></li>
+	<li><a href="news-fr.html" lang="fr">Exemple pratique</a></li>
+</ul>
+
+<h2>Deprecated layout</h2>
+<p>As of version 5.0.0 of GCWeb, the promotions use version 5.0 of Context-specific features. The following code sample is therefore deprecated.</p>
+<pre><code>&lt;aside class="prm-flpr"&gt;
```
Comment (duboisp) [inline templates/news/news-doc-en.html:35]: Ajoute une référence à l'élément précédent, le breadcrumbs.

File: templates/news/news-doc-en.html [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+							&lt;figcaption&gt;
+								&lt;p&gt;Rideau Canal &lt;/p&gt;
+							&lt;/figcaption&gt;
+						&lt;/figure&gt;
+					&lt;/a&gt;
+				&lt;/div&gt;
+			&lt;/div&gt;
+		&lt;/div&gt;
+	&lt;/div&gt;
+&lt;/aside&gt;</code></pre>
```
Comment (duboisp) [inline templates/news/news-doc-en.html:84]: Ajoute une référence à l'élément suivant.

```suggestion
&lt;/aside&gt;
&lt;main>
   ...
&lt;/main></code></pre>
```

File: templates/news/news-doc-fr.html [hunk 6]
```diff
@@ -0,0 +1,87 @@
...
+<ul>
+	<li><a href="news-en.html" lang="en">Working example</a></li>
+	<li><a href="news-fr.html">Exemple pratique</a></li>
+</ul>
+
+<h2>Gabarit obsolète</h2>
+<p>À partir de la version 5.0.0 de GCWeb, les promotions utilisent la version 5.0 des promotions contextuelles. Le morceau de code suivant est donc obsolète.</p>
+<pre><code>&lt;aside class="prm-flpr"&gt;
+	&lt;div class="container"&gt;
+		&lt;h2 class="wb-inv"&gt;Promotions&lt;/h2&gt;
```
Comment (duboisp) [inline templates/news/news-doc-fr.html:37]: Est-ce qu'on avait la version française correspondante?

File: templates/news/news-fr.html [hunk 7]
```diff
@@ -1,13 +1,228 @@
...
+			</figure>
+		</a>
+	</div>
+	<div class="clearfix">
+		<div class="pull-right">
+			{% include components/gc-follow-us/gc-follow-us.html isInline=true %}
+		</div>
+	</div>
+	{% include components/gc-features/features.html %}
+	<div class="row mrgn-tp-md">
```
Comment (duboisp) [inline templates/news/news-fr.html:53]: Ajoute le language attribute

```suggestion
	<div class="row mrgn-tp-md" lang="en">
```

File: templates/topic/topic-testcase-1-en.html [hunk 8]
```diff
@@ -37,7 +37,6 @@ <h3><a href="#">[Subtopic hyperlink text]</a></h3>
 		</div>
 	</section>
 	<div class="col-xs-12 col-md-4">
-		{{>most-requested}}
```
Comment (duboisp) [inline templates/topic/topic-testcase-1-en.html:40]: Met l'include du most requested ici.

File: templates/topic/topic-testcase-1-fr.html [hunk 9]
```diff
@@ -37,7 +37,6 @@ <h3><a href="#">[Lien à un sous-sujet]</a></h3>
 		</div>
 	</section>
 	<div class="col-xs-12 col-md-4">
-		{{>most-requested}}
```
Comment (duboisp) [inline templates/topic/topic-testcase-1-fr.html:40]: Met l'include du most requested ici.

**Review by duboisp — CHANGES_REQUESTED** [review]
Quelques petit changement, voir les notes

File: templates/news/news-doc-fr.html (hunk 6 above)
Comment (Garneauma) [reply templates/news/news-doc-fr.html:41]: Non, malheureusement.

**Review by duboisp — APPROVED** [review]

**Comment by duboisp** [comment]
Reviewed and tested locally, it work as expected and the unused include in the news template are documented.

**Comment by duboisp** [comment]
pre-approve upon final code review of the latest change

**Review by duboisp — APPROVED** [review]
After review, the last change don't impact the previous review I did. It's approved.

**Pull request merged**